src/mount/mount.ceph.c

```diff
@@ -646,12 +658,13 @@ static void modprobe(void)
 
 static void usage(const char *prog_name)
 {
-	printf("usage: %s [src] [mount-point] [-n] [-v] [-o ceph-options]\n",
+	printf("usage: %s [src] [mount-point] [-n] [-v] [-f] [-o ceph-options]\n",
```

Comment:
The problem with this is that mount(8) treats "-f" as "--fake", which basically does everything but the mount syscall. It might not hurt to implement that option too, but repurposing -f like this may cause some confusion.

You may be better off avoiding command-line options for fallback purposes and using a new mount option that's intercepted by the mount helper...or maybe you could turn this into a long-only option (e.g. --force-syntax=v1). Will mount(8) pass through getopt_long options that it doesn't recognize?

Reply:
(Didn't know about fake mount, that's nice)

Looks like we are out of options here - mount(8) does not pass long/short options it does not understand :/

I'll make this as a mount option.